tests/unit/_load_check_required_checks.py

```
@@ -0,0 +1,36 @@
+import importlib.abc
+import importlib.util
+import sys
+from pathlib import Path
+
+_PROJECT_ROOT = Path(__file__).resolve().parents[2]
+_SCRIPT_PATH = _PROJECT_ROOT / "scripts" / "check-required-checks.py"
+
+_MODULE_CACHE = None
+
+
+def load_module():
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `load_module`

(ANN202)